webapp/src/cmds.ts

```diff
@@ -102,13 +102,138 @@ function showUploadInstructionsAsync(fn: string, url: string, confirmAsync: (opt
 
 function hidDeployCoreAsync(resp: pxtc.CompileResult): Promise<void> {
     pxt.debug('HID deployment...');
-    core.infoNotification(lf("Flashing device..."));
+
+    // error message handle in browser download
+    if (!resp.success)
+        return browserDownloadDeployCoreAsync(resp);
+
+    core.infoNotification(lf("Downloading..."));
     let f = resp.outfiles[pxtc.BINARY_UF2]
     let blocks = pxtc.UF2.parseFile(Util.stringToUint8Array(atob(f)))
     return hidbridge.initAsync()
         .then(dev => dev.reflashAsync(blocks))
 }
 
+function askWebUSBPairAsync(resp: pxtc.CompileResult): Promise<void> {
+    const boardName = pxt.appTarget.appTheme.boardName || lf("device");
+    return core.confirmAsync({
+        header: lf("No device detected..."),
+        htmlBody: `
+<p><strong>${lf("Do you want to pair your {0} to the editor?", boardName)}</strong>
+${lf("You will get instant downloads and data logging.")}</p>
+<p class="ui font small">The pairing experience is a one-time process and available for Chrome version 61 and up.</p>
```

Comment:
I certainly wouldn't call out specific version here - if people see it, it means they have the right version. The way you phrase it, they should go and check. I would actually not call out Chrome either, I don't see a point.

Reply:
The point is that they might be confused if they try to use another browser (even on the same machine).

Reply:
At least skip the version number - Chrome auto-updates and usage of older versions is near zero